lib/playground.dart

```diff
@@ -335,6 +332,11 @@ class Playground implements GistContainer, GistController {
     _context.onDartDirty.listen((_) => busyLight.on());
     _context.onDartReconcile.listen((_) => _performAnalysis());
 
+    // Listen for changes that would effect the documentation panel.
+    editor.onMouseDown.listen((e) => docHandler.generateDoc(_docPanel));
+    context.onModeChange.listen((_) => docHandler.generateDoc(_docPanel));
+    context.onModeChange.listen(print);
```

Comment:
Not sure if you forget to remove the debugging message, or if this is on purpose.